skaffold/src/main/kotlin/com/google/container/tools/skaffold/SkaffoldExecutorService.kt

```diff
@@ -43,7 +43,7 @@ abstract class SkaffoldExecutorService {
     }
 
     /** Path for Skaffold executable, any form supported by [ProcessBuilder] */
-    protected abstract var skaffoldExecutablePath: Path
+    open var skaffoldExecutablePath: Path = Paths.get("skaffold")
```

Comment:
this change shouldn't be necessary. the `DefaultSkaffoldExecutorService` is overriding this and setting the value